prow/prowjobs/google/bms-toolkit/test-install-oracle.yaml

```diff
@@ -6,14 +6,11 @@ presubmits:
     decorate: true
     spec:
       containers:
-      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+      - image: quay.io/ansible/ansible-runner:latest
         command:
-        - /bin/bash
-        args:
-        # probing the pod created
-        - "true && id && echo && hostname && echo && pwd && echo && ls -l && echo && whoami && echo && env && echo && df -h"          
+        - ./presubmit_tests/test_pr.sh        
```

Comment:
Any way to qualify this directory?  (Also, where does it come from?  I don't think see a test_pr.sh in this directory)

Reply:
`    decorate: true` directive will clone our bms-toolkit repository **with** the proposed PR and cd into it.
Hence the `.` represents the base directory of the bms-toolkit repo, which is `bms-toolkit`, like:
```
jcnarasimhan@jon2:~/DriveFS/My Drive/bmaas/test_automation/oss_prow_onboarding/running_cleanup_install_oracle$ kubectl exec -it test-pods-namespace-lo-podname -n test-pods -- /bin/bash
bash-4.4# cd /root
bash-4.4# ls -l
total 16
-rw------- 1 root root 2361 Sep 15 14:17 anaconda-ks.cfg
-rw-r--r-- 1 root root  608 Sep 15 14:17 anaconda-post.log
drwxr-xr-x 7 root root 4096 Nov 16 02:13 bms-toolkit
-rw------- 1 root root 2059 Sep 15 14:17 original-ks.cfg
bash-4.4# cd bms-toolkit/
bash-4.4# ls
LICENSE      apply-patch.sh	   check-swlib.sh	config-db.yml		 group_vars	     install-sw.yml	      roles
OWNERS	     asm_disk_config.json  check-swlib.yml	config-rac-db.yml	 host-provision.sh   manage_devices_on_vm.sh  sydney-1.out.thegabba.brutecleanup
README.md    brute-cleanup.yml	   cleanup-oracle.sh	data_mounts_config.json  host-provision.yml  patch.yml
ansible.cfg  check-instance.yml    cluster_config.json	docs			 install-oracle.sh   prep-host.yml
```

I will, in my testing PR, create the ${bms-toolkit-root-dir}/presubmit_tests/test_pr.sh

Reply:
Is it possible to note this in the code itself?  It doesn't seem obvious.  Perhaps a github link to the script itself.  And is it possible to qualify the path to be executed from?  perhaps there might be an environment that already sets this.

Reply:
Sure, adding note to the code.
The script in bms-toolkit repo is non-existent at this point, but will be created when testing this prowjob.
I am not aware of environment variable that prow provides to specify the root directory.

Reply:
Created the <directory>/<.sh> script in a separate PR: https://github.com/google/bms-toolkit/pull/95